src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/dto/ChangelogValuesDTO.kt

```diff
@@ -0,0 +1,26 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.dto
+
+import java.time.OffsetDateTime
+import java.util.UUID
+
+data class ChangelogValuesDTO(
+  val changelogId: UUID,
+  val referralId: UUID,
+  val topic: String,
+  val changedAt: OffsetDateTime,
+  val name: String,
+  val reasonForChange: String
+) {
+  companion object {
+    fun from(changelogValuesDTO: ChangelogValuesDTO): ChangelogValuesDTO {
```

Comment:
I believe this DTO is not necessary. It is creating the ```ChangelogValuesDTO``` from the same Object.It is normally used while mapping the entity object to DTO object

Reply:
Yeah I've removed it now, it wasn't even being used in the controller. Cheers!